docs/network-drivers.md

```diff
@@ -260,3 +260,61 @@ Networks managed by HIL may span multiple switches. No special configuration
 of HIL itself is required; just register each switch as normal and ensure that
 all VLANs in the allocator's ``vlans`` option are trunked to every managed
 switch.
+
+### Openvswitch Driver
+
+This driver is made available so that developers can have real switch
+like functionality without having access to any real switch hardware.
+To get started you will need to:
+1.  install openvswitch in your development machine (or VM).
+
+	yum install openvswitch #For fedora or Centos
+	for i in enable start status
+	do service openvswitch $i; done
+	ovs-vsctl show
+
+2.  create a bridge and name it <switch_name>
+
+	ovs-vsctl add-br <switch_name>
+ 
+3.  Add some ports to this bridge.
+	
+	ovs-vsctl add-port <switch_name> <port_name>
+
+These are just illustrative examples. You may have to do more to setup
+your switch before using it with HIL.
+
+Alternatively you can run a script available at
+
+	https://github.com/SahilTikale/HIL_contrib/blob/master/hilInYourLap/create_datacenter.sh
```

Comment:
I'm not comfortable pointing at this from the official docs unless we're going to go through the work of reviewing it (at which point it would make more sense to just shove it in `examples/`, but it's not necessarily worth the effort). This implies a level of endorsement that I don't think that repository is ready for.

Reply:
We had a discussion may be almost year back, that we should have a contrib directory where we put pointers to the repositories of the contributors where we (HIL developers) can put code for ease of other developers, but do not wish the overhead of getting that reviewed.
The reason for that being two fold:
-- Keeping HIL from bloating that all developers may not want to use.
-- Having developers to add optimizations for specific cases, without the burden on reviewers to go through the burden of reviewing it. 
-- May I should state that all such code that is not blessed as HIL code, will come with no guarantees. 

I put that script just to ease life of other developers. If people have strong opinion against it, I will remove it, but my line of thinking was as described above.

Reply:
Yeah, I remeber that conversation. IIRC, we decided we didn't necessarily need to have the contrib stuff be in the repo at all.

I'm fine with having external resources that are perhaps not quite as vetted, but the way this is worded sounds like a stronger recommendation than I want to see in the official docs.

I wouldn't be opposed to a PR that added some stuff to the docs mentioning the existence your (and possibly other developers') add-on repositories, as long they're clear about the level of endorsement. But I'm not a fan of something like this where there's just a pointer to it in the middle of the general documentation.

Reply:
I have made the wordings explicit that it is optional for reference only.